public_html/wp-content/plugins/pattern-directory/src/pattern-post-type.js

```diff
@@ -11,39 +11,41 @@ import { useDispatch, useSelect } from '@wordpress/data';
 const PatternPostType = () => {
 	const { editPost } = useDispatch( 'core/editor' );
 	const postMetaData = useSelect( ( select ) => select( 'core/editor' ).getEditedPostAttribute( 'meta' ) || {} );
+	const postTitle = useSelect( ( select ) => select( 'core/editor' ).getEditedPostAttribute( 'title' ) || '' );
 	const [ description, setDescription ] = useState( postMetaData.wpop_description );
-	const [ viewportWidth, setViewportWidth ] = useState( postMetaData.wpop_viewport_width );
 
```

Comment:
Do you want to use `usePostData` here?

Reply:
Since it's technically a separate codebase, I decided not to share it. It's not really necessary.... yet.

Reply:
🙃  I was looking through so many files I stopped looking at filepaths. Sounds good.